src/preset_cli/cli/superset/sync/dbt/lib.py

```diff
@@ -459,3 +461,20 @@ def apply_select(
                 del selected[id_]
 
     return list(selected.values())
+
+
+def list_failed_models(failed_models: List[str]) -> None:
+    """
+    List models that failed to sync and ends execution with an error code
+    """
+    error_message = "Below model(s) failed to sync:"
```

Comment:
I thought about adding a redundant check here like this:
``` python
if not failed_models:
    return
```

But I thought it wasn't really necessary since we're only calling this method in case `failed_models` has any value. Let me know if you think we should also add this check here.